### PR TITLE
Get Achievements types from backend

### PR DESCRIPTION
**Pull request by Olya-web7**

### CODE REVIEW

**Review by sivxkxn — APPROVED** [review]
Really good

**Review by Pabloostr — APPROVED** [review]

File: src/app/shared/services/achievements/achievements.service.ts [hunk 1]
```diff
@@ -27,4 +27,8 @@ export class AchievementsService {
   deleteAchievement(id: string): Observable<object> {
     return this.http.delete(`/api/v1/Achievement/Delete/${id}`);
   }
+
+  getAchievementType(): Observable<AchievementType[]> {
```
Comment (litvinets) [inline src/app/shared/services/achievements/achievements.service.ts:31]: getAchievementsType

**Review by litvinets — CHANGES_REQUESTED** [review]
Please, apply it to the form! 
image

**Review by sivxkxn — APPROVED** [review]

**Comment by litvinets** [comment]
image

File: src/app/shell/personal-cabinet/provider/create-achievement/create-achievement.component.ts [hunk 2]
```diff
@@ -78,6 +81,7 @@ export class CreateAchievementComponent implements OnInit, OnDestroy {
       takeUntil(this.destroy$),
       filter((approvedChildren) => !!approvedChildren)
     ).subscribe((approvedChildren: ChildCards) => this.approvedChildren = approvedChildren);
+    this.store.dispatch(new GetAchievementsType());
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/provider/create-achievement/create-achievement.component.ts:84]: mode it to 7 line

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by litvinets — APPROVED** [review]

**Pull request merged**